src/nunit-gui/Program.cs

```diff
@@ -79,7 +80,8 @@ static void Main(string[] args)
             if (options.InternalTraceLevel != null)
                 testEngine.InternalTraceLevel = (InternalTraceLevel)Enum.Parse(typeof(InternalTraceLevel), options.InternalTraceLevel);
 
-            var model = new TestModel(testEngine, options);
+            IMessageDisplay messageDisplay = new MessageDisplay();
+            var model = new TestModel(testEngine, options, messageDisplay);
```

Comment:
I'm afraid I may not have stated the problem clearly enough. It's not that the model should use a platform-independent method to issue messages, it's that it shouldn't be issueing messages at all. Messages are issued by the view, when called by the presenter. The model should not know anything about the view.

Additionally, in the way you implemented it, you didn't really remove the windows dependency because MessageDisplay is part of the SWF implementation.

I'm going to remove this commit, but I wanted to leave you the comments first.